### DISCUSSION TITLE
STM32F429I-DISC1 use of the onboard TFT screen ?

### DISCUSSION THREAD

**Discussion by embsysblog** [micropython/micropython#10128] [Hardware & Peripherals]
I managed to locate the MicroPython binary for the STM32420-DISC1. I'm also able to build it from source. I was wondering if I could access the onboard tft screen in a straightforward fashion. Would I need to include additional libraries (for the LTDC, ili9341 driver and a library like LVGL) before rebuilding the Micropython binary ? I'm still new to Micropython but very curious and keen to learn!

Thanks!

**Comment by peterhinch** [comment]
There are MicroPython drivers for ILI9341. See [nano-gui](https://github.com/peterhinch/micropython-nano-gui) which supports these displays. Also [micro-gui](https://github.com/peterhinch/micropython-micro-gui) which adds user input via pushbuttons.
